404.html

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Let's leave the 404 page for now since if we want to have serve all websites from a server we may want to update the design. Also he 404 needs to updated in server config as well. So let's address this in a separate issue.